head/core/tool_list.cpp

```diff
@@ -0,0 +1,39 @@
+
+
+#include "head/core/tool_list.hpp"
+using namespace ot3_tool_list;
+
+std::array<Tool, 10> _list{
+    {{.tool_type = can_ids::ToolType::pipette_384_chan,
+      .tool_carrier = Z_CARRIER,
+      .bounds = pipette_384_chan_z_bounds},
+     {.tool_type = can_ids::ToolType::pipette_96_chan,
+      .tool_carrier = Z_CARRIER,
+      .bounds = pipette_96_chan_z_bounds},
+     {.tool_type = can_ids::ToolType::pipette_single_chan,
+      .tool_carrier = Z_CARRIER,
+      .bounds = pipette_single_chan_z_bounds},
+     {.tool_type = can_ids::ToolType::pipette_multi_chan,
+      .tool_carrier = Z_CARRIER,
+      .bounds = pipette_multiple_chan_z_bounds},
+     {.tool_type = can_ids::ToolType::pipette_single_chan,
+      .tool_carrier = A_CARRIER,
+      .bounds = pipette_single_chan_a_bounds},
+     {.tool_type = can_ids::ToolType::pipette_multi_chan,
+      .tool_carrier = A_CARRIER,
+      .bounds = pipette_multiple_chan_a_bounds},
+     {.tool_type = can_ids::ToolType::gripper,
+      .tool_carrier = GRIPPER_CARRIER,
+      .bounds = gripper_bounds},
+     {.tool_type = can_ids::ToolType::nothing_attached,
+      .tool_carrier = Z_CARRIER,
+      .bounds = nothing_conected_z_bounds},
+     {.tool_type = can_ids::ToolType::nothing_attached,
+      .tool_carrier = A_CARRIER,
+      .bounds = nothing_conected_a_bounds},
+     {.tool_type = can_ids::ToolType::nothing_attached,
+      .tool_carrier = GRIPPER_CARRIER,
+      .bounds = nothing_conected_gripper_bounds}}};
+auto ot3_tool_list::get_tool_list() -> const std::array<Tool, 10>& {
```

Comment:
`std::array<Tool, 10>`  is a type that is used in various places. 

Could we define an alias? 

Or perhaps use a `std::span` (not sure that is sensible) or a sensible type that doesn't require the size?